qst/src/main/java/io/deephaven/qst/array/DoubleArray.java

```diff
@@ -33,11 +33,11 @@ public static Builder builder(int initialSize) {
         return new Builder(initialSize);
     }
 
-    private static double adapt(Double x) {
+    private static double unbox(Double x) {
```

Comment:
This isn't applied consistently. You did it here, but not in the other primitive arrays. This is the kind of thing that makes me think you really need to use replication.

Reply:
I do like the `box`/`unbox` names better, since they are more descriptive of the transformation.

Reply:
This method doesn't exist anymore. I think everything in Util is consistent.